lib/process-services-cloud/src/lib/process/start-process/components/start-process-cloud.component.spec.ts

```diff
@@ -678,9 +678,11 @@ describe('StartProcessCloudComponent', () => {
             component.ngOnChanges({ appName: firstChange });
             component.processForm.controls['processInstanceName'].setValue('My Process 1');
             component.appName = 'test app name';
+            component.variables = { correlationKey: 'AIyRfpxbBX' };
```

Comment:
where does this key value come from? is it auto-generated randomly on each run?

Reply:
As far as I understand key can be obtain in two ways: one is to get it from @input property variably:any or/and from formCloud (if form is loaded). If it is taken from form it may look like:
 {Text0t9anw: null, Attachfile0tccnd: null, Attachfile08cfo7: null} 
And this data is taken from formCloudService.